### DISCUSSION TITLE
Check anonymization

### DISCUSSION THREAD

**Discussion by ghost** [DNSCrypt/dnscrypt-proxy#1812] [Q&A]
Hi,
Having set up DNScrypt with anonymized DNS, I was wondering whether there is a way to check that anonymization actually works. I use dnslesktest.com to make check that DNS is encrypted, but that does not indicate anonymization. 
In particular, the warning "[NOTICE] Anonymized DNS: routing everything via [anon-XXX anon-XXX]" indicated at the bottom of https://github.com/DNSCrypt/dnscrypt-proxy/wiki/Anonymized-DNS does not show up for me, leading me to think I may not have configured this properly.
Any ideas? Thanks!
˜Paul

**Comment by jedisct1** [comment]
A notice message such as the one you mentioned should be displayed.

But another way to check is to use `scaleway-fr` or `scaleway-ams` as a server, and do a query for `my.ip`.

If this returns your real IP address, there's no anonymization. If you get the relay address, anonymization is working.

**Reply by ghost** [reply]
Ok, so to be honest, I am not sure if I did the query right. But looking up my IP online does provide my real IP, which means the anonymization is not happening, right? Do the servers and relay need to be defined in the static section at the bottom? (I did that just in case, but the result is the same).
Thanks in advance!
˜Paul

**Reply by ghost** [reply]
Hi @jedisct1 , any idea why the warning was not displayed in the first place, perhaps? Any help is useful!
